RELEASE-PLAN.md

```diff
@@ -0,0 +1,53 @@
+# Background and Context
+
+Releases of the Pathfinder Framework (Version 2 can be found [here](https://www.carbon-transparency.com/media/jpslsujn/pathfinder-framework.pdf)) are strongly
+informing the tech specs releases.
+
+Therefore, this timeline page contains the release plan for both, the Pathfinder Framework and the Pathfinder Tech Specs.
+
+In addition, the tables below disclose deprecation of releases and subsequent sunset dates.
+
+## Definitions
+
+<dl>
+<dt>Draft Release</dt>
+<dd>The release of draft document, starting a period of feedback collection from the community</dd>
+<dt>Deprecation</dt>
+<dd>Companies are no longer encouraged to use the release (e.g. to compute and subsequently publish PCFs based on it)</dd>
+<dt>Sunset</dt>
+<dd>The release is no longer supported. Companies are discouraged to use the release (e.g. to compute and to subsequently publish PCFs based on it)</dd>
+</dl>
+
+
+
+**As always, the timelines are subject to change.**
+
+## Pathfinder Framework: Upcoming Releases
+
+| When | Milestone | What |
+| --- | --- | --- |
+| December 31, 2023 | Deprecation | Pathfinder Framework v1 |
+| Mid-May to Mid-June 2024 | Draft Release | Pathfinder Framework v2.1 |
```

Comment:
Joy recommended to add a caveat to the Draft Release date (see https://www.notion.so/2024-PACT-Timelines-adf4cdeb48934a0cac61fe54c59e208c) because the biogenic chapter will most likely be delayed. Up to you how you think best to format that.

Reply:
added in 54d472659c19e7d5a480af0b5a8a59a292f772a7